### PR TITLE
Adjust Language and Direction Capability

### PR DESCRIPTION
**Pull request by amiroperator**
Adjust persian language
Add direction capability to make template better for right to left languages

### CODE REVIEW

**Comment by LuckyCyborg** [comment]
The key 'name' in the Language list should be the localized name of the Language. i.e Persian in Persian

File: app/Config.example.php [hunk 1]
```diff
@@ -115,7 +115,7 @@
     'de' => array('info' => 'German',    'name' => 'Deutsch',    'locale' => 'de_DE'),
     'en' => array('info' => 'English',   'name' => 'English',    'locale' => 'en_US'),
     'es' => array('info' => 'Spanish',   'name' => 'Español',    'locale' => 'es_ES'),
-    'fa' => array('info' => 'Persian',   'name' => 'Persian',    'locale' => 'fa_FA'),
+    'fa' => array('info' => 'Persian',   'name' => 'Persian',    'locale' => 'fa_IR'),
     'fr' => array('info' => 'French',    'name' => 'Français',   'locale' => 'fr_FR'),
     'it' => array('info' => 'Italian',   'name' => 'italiano',   'locale' => 'it_IT'),
     'ja' => array('info' => 'Japanesse', 'name' => 'Japanesse',  'locale' => 'ja_JA'),
```
Comment (Kingorgg) [inline app/Config.example.php]: @LuckyCyborg So, would:
````php
'ja' => array('info' => 'Japanese', 'name' => 'Japanese',  'locale' => 'ja_JA'),
````
Need to be changed to:
````php
'ja' => array('info' => 'Japanese', 'name' => '日本語',  'locale' => 'ja_JA'),
```

**Comment by LuckyCyborg** [comment]
@Kingorgg Yep

**Comment by Kingorgg** [comment]
@amiroperator Could you make the change for the Japanese language in your update, please?

**Comment by LuckyCyborg** [comment]
@amiroperator Hey, why you removed "css/style.css" from "custom.php" Layout?

File: app/Config.example.php (hunk 1 above)
Comment (LuckyCyborg) [inline app/Config.example.php]: yep

**Comment by amiroperator** [comment]
Wait please I'll fix them

**Comment by amiroperator** [comment]
@LuckyCyborg I didn't remove  "css/style.css" from  from "custom.php" Layout, I just adjust them for different directions

File: app/Config.example.php (hunk 1 above)
Comment (amiroperator) [inline app/Config.example.php]: Updated

**Comment by LuckyCyborg** [comment]
@amiroperator Another thing. While you have good intentions, the logic to calculate the direction is flawed because the Language can be changed on-fly.

Please make Language::direction() as following, per moment:

```
public static function direction($code = LANGUAGE_CODE)
{
    return 'ltr';
}
```
And I will complete myself the logic there, after this PR is merged.

**Comment by amiroperator** [comment]
@LuckyCyborg  Ok.

**Comment by Kingorgg** [comment]
@amiroperator Thank You!

**Comment by LuckyCyborg** [comment]
@amiroperator I suppose that you understand that all those changes will work **only** on the new Language API, right?

**Comment by amiroperator** [comment]
I closed this because I want to make this again

**Comment by LuckyCyborg** [comment]
OK

**Pull request closed without merging**